proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -27,11 +25,8 @@ message EventPut {
   // basket_denom is the basket bank denom that the credits were added to.
   string basket_denom = 2;
 
-  // credits are the credits that were added to the basket.
-  repeated BasketCredit credits = 3;
```

Comment:
We are not bumping the basket proto version in this release so we may need to deprecate and add rather than updating the existing messages and fields:

https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-044-protobuf-updates-guidelines.md

We started using `Since Revision 1` for fields we are adding rather than what would be `Since: v4.0` based on the guidelines linked above. Let's discuss this on our call tomorrow.

Reply:
Based on our discussion, we should keep the field and mark it as deprecated and remove in the next version.

Reply:
reverted https://github.com/regen-network/regen-ledger/pull/1044/commits/4e607ee07f8cf91e96cbc564bf5dca0df6f94cbd

Reply:
> and mark it as deprecated

Can we market it as deprecated? :pray:

Reply:
woops, added: https://github.com/regen-network/regen-ledger/pull/1044/commits/99bf0286a9492869793ef77ee2540424a1686996